### PR TITLE
Add Logs to ch36 Workflow

### PR DESCRIPTION
**Pull request by DLarson-Oddball**
## Summary

Adds logs to ch36 workflow creating team visibility into submission failure and success rates.
Adds log to ch31 case where participant_id is blank.

## Related issue(s)
- [ department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67879)

## Testing done

- Tested flow using debugger to ensure code runs correctly.

## What areas of the site does it impact?
Ch36 submission background job

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: app/sidekiq/central_mail/submit_career_counseling_job.rb [hunk 1]
```diff
@@ -3,13 +3,21 @@
...
     def perform(claim_id, user_uuid = nil)
-      @claim = SavedClaim.find(claim_id)
-      @claim.send_to_central_mail!
-      send_confirmation_email(user_uuid)
+      begin
+        @claim = SavedClaim.find(claim_id)
+        @claim.send_to_central_mail!
+        send_confirmation_email(user_uuid)
+      rescue => e
+        log_message_to_sentry('Error submitting form 25-8832', :warn, { uuid: user_uuid })
```
Comment (micahaspyr) [inline app/sidekiq/central_mail/submit_career_counseling_job.rb:16]: Change :warn to :error

**Review by micahaspyr — CHANGES_REQUESTED** [review]
Only suggestion is to change :warn to :error

**Review by micahaspyr — APPROVED** [review]
LGTM

**Review by tblackwe — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**